PROTOCOL.md

```diff
@@ -2807,6 +2905,108 @@ nullCount | The number of `null` values for this column
 minValues | A value smaller than all values present in the file for this column
 maxValues | A value larger than all values present in the file for this column
 
+## API Response Format in Delta
+This section discusses the API Response Format in Delta returned by the server.
```

Comment:
Perhaps comments on the overall motivation here (to make this section self contained).

Reply:
updated.